Vagrantfile

```diff
@@ -9,26 +9,31 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
 
-  # Create port forward mappings
-  # These are not normally required, since all access happens on the
-  # 192.168.77.77 IP address
-  #
-  # config.vm.network "forwarded_port", guest: 80, host: 80
-  # config.vm.network "forwarded_port", guest: 443, host: 443
-  # config.vm.network "forwarded_port", guest: 4443, host: 4443
-  # config.vm.network "forwarded_port", guest: 8501, host: 8501
-
   vm_memory = ENV.fetch('VM_MEMORY', 10 * 1024).to_i
   vm_cpus = ENV.fetch('VM_CPUS', 4).to_i
 
+  os=`uname`.strip
+  if os == 'Darwin'
+    default_if=`route get default | grep interface`.split(" ").last
+  else
+    default_if=`/sbin/route | grep default`.split(" ").last
+  end
+  # Ugly hack to warn user about not using a host bridge with libvirt
+  user_warned_about_bridge = false
+
+  net_config = {
+    :use_dhcp_assigned_default_route => true
```

Comment:
Thanks, I forgot ruby did this magic conversion under-the-hood for configuration hashes for all hash expressions/declarations

But this isn't relevant to the current state, since there was no network configuration common to all deployments in the end.